modules/PhyVersoBSP/phyverso_mcu_comms/evSerial.h

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Pionix GmbH and Contributors to EVerest
+
+#ifndef YETI_SERIAL
+#define YETI_SERIAL
```

Comment:
looks like a copy-paste error and should probaby be something like PHYVERSO_MCU_COMMS_SERIAL